tests/test_formatters_as_dataframe.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import print_function
+
+import pytest
+pd = pytest.importorskip('pandas')
+import numpy as np
+
+from eli5 import format_as_dataframes, format_as_dataframe, format_as_text
+from eli5.base import (
+    Explanation, TargetExplanation, FeatureWeight, FeatureWeights,
+    FeatureImportances, TransitionFeatureWeights,
+)
+
+
+@pytest.mark.parametrize(
+    ['with_std', 'with_value'],
+    [[False, False], [True, False], [False, True]])
+def test_targets(with_std, with_value):
+    expl = Explanation(
+        estimator='some estimator',
+        targets=[
+            TargetExplanation(
+                'y', feature_weights=FeatureWeights(
+                    pos=[FeatureWeight('a', 13,
+                                       std=0.13 if with_std else None,
+                                       value=2 if with_value else None),
+                         FeatureWeight('b', 5,
+                                       std=0.5 if with_std else None,
+                                       value=1 if with_value else None)],
+                    neg=[FeatureWeight('neg1', -10,
+                                       std=0.2 if with_std else None,
+                                       value=5 if with_value else None),
+                         FeatureWeight('neg2', -1,
+                                       std=0.3 if with_std else None,
+                                       value=4 if with_value else None)],
+                )),
+            TargetExplanation(
+                'y2', feature_weights=FeatureWeights(
+                    pos=[FeatureWeight('f', 1)],
+                    neg=[],
+                )),
+        ],
+    )
```

Comment:
Agreed, that would be much more robust. I added most tests in b4bc427, going to add CRF checks right in existing CRF tests - they show a failure during export by the way :)